pipelines/helfi-redirector-dev.yml

```
@@ -0,0 +1,49 @@
+# Continuous integration (CI) triggers cause a pipeline to run whenever you push
+# an update to the specified branches or you push specified tags.
+trigger:
+  batch: true
+  branches:
+    include:
+      - main
+  paths:
+    exclude:
+    - '*release-please*'
+    - '*.md'
+    - '.github/'
+    - 'pipelines/helfi-redirector-release.yml'
+    - 'pipelines/helfi-redirector-review.yml'
+    - 'docker-compose*'
```

We use compose.yaml format for the compose file ([as recommended by Docker](https://docs.docker.com/compose/intro/compose-application-model/#the-compose-file))

```suggestion
    - 'compose*'
```

---

fixed